tests/foreman/ui/test_host.py

```diff
@@ -372,6 +359,37 @@ def module_libvirt_hostgroup(
     ).create()
 
 
+@pytest.fixture(scope='module')
```

Comment:
Hi, I see there is a `def module_manifest_org():` fixture in `pytest_fixtures/api_fixtures.py` you could try.

Reply:
At this time we're not necessarily enforcing use of the common fixtures, but it is highly encouraged to not define an equivalent entity local to the test module.

I wouldn't block merging this PR with this definition here, but this fixture is identical to the one @swadeley has pointed out.

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L55

Given pytest fixture resolution, @pondrejk this wouldn't require any changes other than simply deleting L362-366. `manifest_org` will still be resolved by `module_activation_key`.